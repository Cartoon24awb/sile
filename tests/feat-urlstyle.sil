\begin[papersize=a6]{document}
\neverindent
\nofolios
\use[module=packages.url]

\url{https://sile-typesetter.org/}

\define[command=urlstyle]{<{\process}>}
\url{https://sile-typesetter.org/}

\end{document}
