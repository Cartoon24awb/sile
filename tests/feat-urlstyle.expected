Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	26.2947
Set font 	Hack;7;400;;normal;;;LTR
T	156 211 211 198 205 w=21.0718	(https)
Mx 	35.9537
T	583 w=4.2144	(:)
Mx 	40.1680
T	599 w=4.2144	(/)
Mx 	44.3824
T	599 w=4.2144	(/)
Mx 	48.5967
T	205 158 172 139 w=16.8574	(sile)
Mx 	65.4542
T	656 w=4.2144	(-)
Mx 	69.6685
T	211 233 198 139 205 139 211 211 139 201 w=42.1436	(typesetter)
Mx 	111.8121
T	592 w=4.2144	(.)
Mx 	116.0264
T	184 201 151 w=12.6431	(org)
Mx 	128.6695
T	599 w=4.2144	(/)
Mx 	14.8819
My 	38.2947
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	31 w=4.3896	(<)
Mx 	19.2715
T	75 87 87 83 86 w=21.5625	(https)
Mx 	40.8340
T	29 w=2.2900	(:)
Mx 	43.1241
T	18 w=4.6924	(/)
Mx 	47.8165
T	18 w=4.6924	(/)
Mx 	52.5088
T	86 76 79 72 w=13.9014	(sile)
Mx 	66.4102
T	16 w=3.3691	(-)
Mx 	69.7794
T	87 92 83 72 86 72 87 87 72 85 w=42.2363	(typesetter)
Mx 	112.0157
T	17 w=2.2900	(.)
Mx 	114.3057
T	82 85 74 w=13.8672	(org)
Mx 	128.1729
T	18 w=4.6924	(/)
Mx 	132.8653
T	33 w=4.3896	(>)
End page
Finish
